### PR TITLE
Fix: Ensure wallet1 is using the faucet privkey even if web isn't deployed

### PR DESCRIPTION
**Pull request by ktechmidas**
## Issue being fixed or feature implemented
Moved importing the privkey for the faucet to the main section of dashd so can make multifaucet optional. It makes no sense that the whole mining wallet that the mn-init role relies on is done in multifaucet of all roles

## What was done?
Import privkey into new wallet1 before continuing setup

## How Has This Been Tested?
devnet-infra-argh

## Breaking Changes
None

### CODE REVIEW

File: ansible/roles/dashd/tasks/main.yml [hunk 1]
```diff
@@ -96,5 +96,9 @@
   retries: 5
   delay: 10
 
+- name: import faucet privkey
+  when: inventory_hostname in groups['faucet_wallet']
+  shell: dash-cli importprivkey {{ faucet_privkey }}
```
Comment (strophy) [inline ansible/roles/dashd/tasks/main.yml:101]: Are you sure this works before the wallet is created?

**Comment by strophy** [comment]
Good catch! Why do we need to run it twice, before and after the wallet is created?

File: ansible/roles/dashd/tasks/main.yml (hunk 1 above)
Comment (ktechmidas) [reply ansible/roles/dashd/tasks/main.yml:104]: Oh that's a good catch, I've moved it below the "create_wallet" section

Comment (ktechmidas) [reply ansible/roles/dashd/tasks/main.yml:104]: So it should work on both <18.2 and >18.2 now

**Review by strophy — APPROVED** [review]
:+1: good change

**Pull request merged**